mappings/net/minecraft/client/performance/MetricSupplier.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_5948 net/minecraft/client/performance/MetricSupplier
+	FIELD field_29548 metric Lnet/minecraft/class_5951;
+	FIELD field_29549 supplier Ljava/util/function/DoubleSupplier;
+	FIELD field_29550 type Lnet/minecraft/class_5949;
+	METHOD <init> (Lnet/minecraft/class_5951;Ljava/util/function/DoubleSupplier;Lnet/minecraft/class_5949;)V
+		ARG 1 metric
+		ARG 2 supplier
+		ARG 3 type
+	METHOD method_34697 getMetric ()Lnet/minecraft/class_5951;
+	METHOD method_34698 getSupplier ()Ljava/util/function/DoubleSupplier;
```

Comment:
Hmm, `supplier.getSupplier` sounds a bit weird. What value does this supplier obtain?

Reply:
Double

Reply:
Hmm, this naming setup is a bit weird. Allow me to come back later, a bit busy recently.